runtime/sema/check_composite_declaration.go

```diff
@@ -1977,6 +1996,120 @@ func (checker *Checker) enumMembersAndOrigins(
 	return
 }
 
+func (checker *Checker) checkDefaultDestroyParamExpressionKind(
+	arg ast.Expression,
+) {
+	switch arg := arg.(type) {
+	case *ast.StringExpression,
+		*ast.BoolExpression,
+		*ast.NilExpression,
+		*ast.IntegerExpression,
+		*ast.FixedPointExpression,
+		*ast.PathExpression:
+		break
+	case *ast.IdentifierExpression:
```

Comment:
Here and below: Add whitespace to separate cases, to make the switch easier to read

```suggestion

	case *ast.IdentifierExpression:
```